pkg/kubelet/eviction/api/types.go

```diff
@@ -40,6 +40,13 @@ const (
 	SignalAllocatableMemoryAvailable Signal = "allocatableMemory.available"
 	// SignalPIDAvailable is amount of PID available for pod allocation
 	SignalPIDAvailable Signal = "pid.available"
+
+	// SignalEphemeralContainerFsLimit is amount of storage available on filesystem requested by the container
+	SignalEphemeralContainerFsLimit Signal = "ephemeralcontainerfs.limit"
```

Comment:
Signals are generally used for specifying node-level limits, e.g. with `--eviction-hard=pid.available<1000`.  I think it would be better to define these constants (and they don't have to have the form `foo.bar`) in the eviction package.

Reply:
Ok

Reply:
I think keeping it reasonable consistent makes the metric make more sense - updated, PTAL